packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -137,9 +111,7 @@ class PropertyName extends StatelessWidget {
       padding: const EdgeInsets.all(denseRowSpacing),
       child: Text(
         property.name ?? '',
-        style: DiagnosticsTextStyles.regular(
-          theme.colorScheme,
-        ).merge(theme.subtleTextStyle),
+        style: theme.subtleTextStyle,
```

Comment:
super-nit: now there is no need to store the theme in a var because it is only looked up once.

Reply:
Done!